core/src/main/scala/org/apache/spark/deploy/master/WorkerInfo.scala

```diff
@@ -107,4 +107,6 @@ private[spark] class WorkerInfo(
   def setState(state: WorkerState.Value): Unit = {
     this.state = state
   }
+
+  def isAlive(): Boolean = this.state == WorkerState.ALIVE
```

Comment:
Hi Andrew

Same reason as my comment. I think this is only way we can easily add one line code without too much trouble